recipes/qt-async-threads/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qt-async-threads" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qt-async-threads-{{ version }}.tar.gz
+  sha256: 1cec7374336e3925344055ca5cb191d95329c90edccbea2c0d4ecdfa75854e8c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.10
+    - setuptools
+    - setuptools-scm
+    - toml
+  run:
+    - attrs
+    - boltons
+    - pyqt >=5.12
+    - python >=3.10
+
+test:
+  imports:
+    # skipping importing tests due to some problem with conda-forge:
+    # libGL.so.1: cannot open shared object file: No such file or directory
+    # - qt_async_threads
```

Comment:
Add yum_requirements.txt for the missing library.

https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl

Reply:
Thanks, TIL, done!